eureka-service-discovery/README.md

```diff
@@ -0,0 +1,12 @@
+# Eureka Discovery service
+## Application provides service discovery mechanism - each service knows the addresses of existing Eureka instances and 
+sends some heartbeats to it periodically. So long as Eureka receives these heartbeats from service it is considered to be 
```

Comment:
AS long as Eureka receives? I'm not sure in "so long as" https://dictionary.cambridge.org/ru/%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/%D0%B1%D1%80%D0%B8%D1%82%D0%B0%D0%BD%D1%81%D0%BA%D0%B0%D1%8F-%D0%B3%D1%80%D0%B0%D0%BC%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0/as-and-as-expressions/as-long-as-and-so-long-as

Reply:
that means condition. but ok, there is no difference for me :)